js/src/components/paid-ads/assetSpecs.js

```diff
@@ -260,56 +271,122 @@ const ASSET_TEXT_SPECS = [
 // functions inside are for initialization purposes only and are not expected to be exposed
 // outside this module.
 {
-	function getSubheading( min, max ) {
+	function getSubheading( spec, shownAsSharedMax ) {
+		if ( shownAsSharedMax ) {
```

Comment:
❓ I don't understand well this `shownAsSharedMax` since we are not handling MAX but yes only MIN label, right?  ... Shouldn't be `showOnlyMinLabel` or something like that? 

Another option would be a negation of `showMaxLabel` 

`if ( ! showMaxLabel ) { `

Also, I take as granted that there is always a `spec.max` defined.

Reply:
This is 💅  not a blocker

Reply:
Ok. I see this name is potentially coming from https://github.com/woocommerce/google-listings-and-ads/pull/1894 shared image max limits. Makes sense now.

Reply:
Sidenote: The reason is also shown here:
https://github.com/woocommerce/google-listings-and-ads/blob/b9b6b451dd593fba975c3eb5b0b551190bdfece8/js/src/components/paid-ads/assetSpecs.js#L375-L379

So there are three different copies on UI:

![2023-02-22 16 36 52](https://user-images.githubusercontent.com/17420811/220568266-02b2ff2c-ea3e-450a-8c40-a5d4408f5b72.png)